README.md

```diff
@@ -44,10 +46,7 @@ rails g rolify Role User
 rails g rolify:role Role User
 ```
 
-Role and User classes are the default. You can specify any Role class name you want. This is a completely new file so any name will do the job.
-For the User class name, you would probably use the one provided by your authentication solution. rolify just adds some class methods in an existing User class.
-
-If you want to use Mongoid instead of ActiveRecord, just add `--orm=mongoid` argument, and skip to step #3
+The generator will create your Role model, add a migration file, and update your User class with new class methods. 
```

Comment:
The summary of what the generator does helps the user understand more of what is going on.